SPECS/igc-module.spec

```diff
@@ -0,0 +1,54 @@
+%define uname  %{kernel_version}
+%define module_dir updates
```

Comment:
module_dir should be extra, here, as we don't update anything: https://xcp-ng.org/docs/develprocess.html#where-the-modules-are-installed-on-the-system